paper-tooltip.html

```diff
@@ -70,6 +70,7 @@
         outline: none;
         @apply(--paper-font-common-base);
         font-size: 10px;
+        line-height: 10px;
```

Comment:
We can also use `line-height: 1;` so if `font-size` changes `line-height` would have the same value.